src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/DrawingView.shared.cs

```diff
@@ -94,5 +97,19 @@ static object CoerceValue(BindableObject bindable, object value)
 			Color strokeColor,
 			Color backgroundColor) =>
 			DrawingViewService.GetImageStream(points.ToList(), imageSize, lineWidth, strokeColor, backgroundColor);
+
+		public void OnDrawingCompleted()
```

Comment:
I think it should be internal

Reply:
I did originally make it `internal` but the `DrawingViewRenderer.wpf.cs` class is external to this project.

Reply:
DrawingViewRenderer.wpf.cs is a sample. and not a part of the plugin, so let's make this method internal

Reply:
Oh I completely missed that! I can revert the renderer code to use the Command implementation then.

Reply:
This should now be addressed